lib/aff4_objects/security.py

```diff
@@ -151,10 +151,11 @@ class ApprovalWithApproversAndReason(Approval):
 
   class SchemaCls(Approval.SchemaCls):
     """The Schema for the ClientAccessApproval class."""
+
     LIFETIME = aff4.Attribute(
         "aff4:approval/lifetime", rdfvalue.RDFInteger,
-        "The number of microseconds an approval is valid for.",
-        default=4 * 7 * 24 * 60 * 60 * 1000000)  # 4 weeks
+        "The number of seconds an approval is valid for.",
```

Comment:
I agree that config_lib.CONFIG["ACL.token_expiry"] should override the default hard coding but I was seeing an exception thrown that config_lib.CONFIG["ACL.token_expiry"] was not loaded yet. A sequence/timing issue ? So, in CheckAccess(), I removed the reference to LIFETIME and used the config_lib variable instead since it has a default value defined in server.py. If we still want to use the LIFETIME attribute, then I need to resolve the original exception.